### DISCUSSION TITLE
Intercepting ref with <Controller />

### DISCUSSION THREAD

**Discussion by chardypilot** [react-hook-form/react-hook-form#4513] [Q&A]
Hello, 

My issue is similar to this https://github.com/react-hook-form/react-hook-form/issues/25 but with the "Controller" element instead.
When I pass ref to the controlled element, handleSubmit doesn't get the input data anymore.
I'm using with Ionic react

Thanks beforehand for your help and insight.
`<Controller control={control} name="email" render={({onBlur, onChange, value, ref})=>( <IonInput color="dark" onIonBlur={onBlur} onIonChange={onChange} value={value} mode="md" autocomplete="email" clearInput={true} ref={ref} type="email" inputmode="email" /> )} />
`

**Comment by jorisre** [comment]
Hi @chardypilot 

Can you share a codesandbox with your problem please ?
